www/js/pedagogical_module.js

```diff
@@ -197,20 +231,17 @@ define([
 				}else{
 					givenID = this.model.student.getAuthoredID(id);
 					console.assert(nodeEditorActionTable[interpretation], "processAnswer() interpretation '" + interpretation + "' not in table ", nodeEditorActionTable);
-					nodeEditorActionTable[interpretation][this.userType](returnObj, nodePart);
-					
+					nodeEditorActionTable[interpretation][this.userType](returnObj, nodePart, answer);
```

Comment:
where is this thing? I mean the nodeEditorActionTable. Why cant I see it where it is defined.